Magazine/Controllers/HomeController.cs

```diff
@@ -1,5 +1,7 @@
 ﻿namespace Magazine.Controllers;
-public class HomeController : Controller
+
+[AllowAnonymous]
```

Comment:
this is should be the case any way, one of the downsides of having a base controller that only suits some